harness/determined/cli/_color_json.py

```diff
@@ -0,0 +1,92 @@
+"""
+colorize JSON output.
+test string:
+   echo '{"a": "b", "c": ["1", 2.0, 3.1, 5, "4", {"d": true, "e": null, "f": [1, 2, 3]}]}'
+"""
+
+import json
+from typing import Any, TextIO
+
+from termcolor import colored
+
+KEY = "blue"
```

Comment:
nit: I'm not sure the singular/plural distinction between string, key, primitives, and separators was intentional.

Reply:
I was going with: key applies to one type of str we color but the others apply to multiple types 🤷🏼